.github/workflows/test_install.yml

```
@@ -0,0 +1,38 @@
+name: 'test install'
+
+on:
+  push:
+    branches:
+      - "dev"
+  pull_request:
+    branches:
+      - '*'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test_install:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest", "macos-latest"]
+        python-version: ["3.8", "3.9", "3.10"]
+    name: ${{ matrix.os }} with Python ${{ matrix.python-version }}
```

We will likely add more GH worflows in the future. This will help distinguishing them within the jobs tab.

```suggestion
    name: Install on ${{ matrix.os }} with Python ${{ matrix.python-version }}
```